content/introduction/overview.md

```diff
@@ -30,57 +34,57 @@ Making it possible to download a file from many locations that aren’t managed
+It’s important to remember in all of these situations using IPFS is participatory and collaborative. If nobody using IPFS has the content identified by a given address available for others to access, you won’t be able to get it. On the other hand, content can’t be removed from IPFS as long as *someone* is interested enough to make it available, whether that person is the original author or not.
 
 
 ## It’s all about possession and participation.
 
-While there is lots of complex technology in IPFS, the fundamental ideas are about changing how networks of people and computers communicate: while today’s world-wide-web is structured on *ownership* and *access*, where you get files from whoever owns them and they choose to grant you access, IPFS is based on the ideas of *possession* and *participation*, where many people have each others’ files and *participate* in making them available.
+While there’s lots of complex technology in IPFS, the fundamental ideas are about changing how networks of people and computers communicate. Today’s World Wide Web is structured on *ownership* and *access*, meaning that you get files from whoever owns them — if they choose to grant you access. IPFS is based on the ideas of *possession* and *participation*, where many people have each others’ files and *participate* in making them available.
 
-That means IPFS only works well when people are actively participating. If you share a file on your computer, but then turn your computer off, other people won’t be able to access it from you. But if you or others make sure a copy of that file is stored on another computer running IPFS, it will be available more often. By default, your computer participates in sharing a file with others for a limited time after you’ve downloaded it with IPFS. You can *pin* a file to save and permanently share it.
+That means IPFS only works well when people are actively participating. If you use your computer to share files using IPFS, but then you turn your computer off, other people won’t be able to get those files from you anymore. But if you or others make sure that copies of those files are stored on more than one computer that’s powered on and running IPFS, those files will be more reliably available to other IPFS users who want them. This happens to some extent automatically: by default, your computer shares a file with others for a limited time after you’ve downloaded it using IPFS. You can also make content available more permanently by *pinning* it, which saves it to your computer and makes it available on your IPFS network until you decide to *unpin* it. (You can learn more about this in the [concept guide on pinning]({{<relref "guides/concepts/mfs.md">}})!)
```

Comment:
note content is made available to the IPFS network (not `your IPFS network`) without being pinned, as long as it is downloaded. Pinning just prevents content from being removed when GCing. You can see pins as "favourites" which stay after you clean your browsing history.

Reply:
`your` --> `the` in e61efaf

I believe the existing paragraph already covers the download/share versus pin/more permanently share situation, at a high level.